orbit_determination/ekf.py

```diff
@@ -46,13 +51,15 @@ def __init__(
         # :param w_b: Initial angular velocity bias with shape (3,)
         :param P: Initial covariance with shape (9, 9)
         :param Q: Process noise covariance with shape (16, 16)
-        :param R: Measurement noise covariance with shape (3, 3)
+        :param R_vec: Measurement noise covariance with shape depending on the number of landmarks
         :param dt: The amount of time between each time step.
+        :param w: The angular velocity of the satellite with shape (3,)
 
         :return: None
 
-        Note on R matrix dimensionality: As the number of landmarks observed will change between individual time steps,
-        the R matrix needs to be constructed at each time step where the vision pipeline is used.
+        Note on R_vec matrix dimensionality: As the number of landmarks observed will change between
```

Comment:
It would be nice to explain how the dimensionality of R depends on the number of landmarks observed (i.e. shape (n, n)? or whatever it is).